managed/provider.go

```diff
@@ -155,6 +155,7 @@ func (p *provider) GetResources(_ context.Context) (map[string]tfsdk.ResourceTyp
 		"ybm_api_key":                            resourceApiKeyType{},
 		"ybm_metrics_exporter":                   resourceMetricsExporterType{},
 		"ybm_associate_metrics_exporter_cluster": resourceAssociateMetricsExporterClusterType{},
+		"ybm_cluster_db_audit_export_config":     resourceClusterDbAuditExportConfigType{},
```

Comment:
if you associate a config to a cluster then I think `ybm_associate` make more sense.

Reply:
Fixed